### ISSUE TITLE
when run lower_example.py the Accurary is None, i didn't modify anythings

### ISSUE THREAD

**Issue by cqray1990** [pytorch/TensorRT#1748]
## To Reproduce

1.
1.
1.

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version (e.g. 1.0.0):
 - PyTorch Version (e.g. 1.0):1.13.1
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:3.8
 - CUDA version:CUDA 11.7
 - GPU models and configuration: 1060
 - Any other relevant information:

## Additional context

=== Running benchmark for: Configuration(batch_iter=50, batch_size=128, name='CUDA Eager', trt=False, jit=False, fp16=False, accuracy_rtol=-1) green
== Start benchmark iterations
== End benchmark iterations
=== Running benchmark for: Configuration(batch_iter=50, batch_size=128, name='TRT FP32 Eager', trt=True, jit=False, fp16=False, accuracy_rtol=0.001) green
INFO:torch_tensorrt.fx.passes.pass_utils:== Log pass <function fuse_permute_matmul at 0x7ff2e1c9c4c0> before/after graph to /tmp/tmpaes5m2xh, before/after are the same = True, time elapsed = 0:00:00.933102
INFO:torch_tensorrt.fx.passes.pass_utils:== Log pass <function fuse_permute_linear at 0x7ff2e1c9c280> before/after graph to /tmp/tmp2y9uxq3_, before/after are the same = True, time elapsed = 0:00:00.494979

Supported node types in the model:
acc_ops.conv2d: ((), {'input': torch.float32, 'weight': torch.float32})
acc_ops.batch_norm: ((), {'input': torch.float32, 'running_mean': torch.float32, 'running_var': torch.float32, 'weight': torch.float32, 'bias': torch.float32})
acc_ops.relu: ((), {'input': torch.float32})
acc_ops.max_pool2d: ((), {'input': torch.float32})
acc_ops.add: ((), {'input': torch.float32, 'other': torch.float32})
acc_ops.adaptive_avg_pool2d: ((), {'input': torch.float32})
acc_ops.flatten: ((), {'input': torch.float32})
acc_ops.linear: ((), {'input': torch.float32, 'weight': torch.float32, 'bias': torch.float32})

Unsupported node types in the model:

Got 1 acc subgraphs and 0 non-acc subgraphs
INFO:torch_tensorrt.fx.passes.lower_pass_manager_builder:Now lowering submodule _run_on_acc_0
INFO:torch_tensorrt.fx.lower:split_name=_run_on_acc_0, input_specs=[InputTensorSpec(shape=(-1, 3, 224, 224), dtype=torch.float32, device=device(type='cuda', index=0), shape_ranges=[((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224))], has_batch_dim=True)]
INFO:torch_tensorrt.fx.lower:Timing cache is used!
INFO:torch_tensorrt.fx.fx2trt:TRT INetwork construction elapsed time: 0:00:00.756111
INFO:torch_tensorrt.fx.fx2trt:Build TRT engine elapsed time: 0:00:38.111414
INFO:torch_tensorrt.fx.passes.lower_pass_manager_builder:Lowering submodule _run_on_acc_0 elapsed time 0:00:44.544102
== Start benchmark iterations
== End benchmark iterations
=== Running benchmark for: Configuration(batch_iter=50, batch_size=128, name='TRT FP16 Eager', trt=True, jit=False, fp16=True, accuracy_rtol=0.01) green
INFO:torch_tensorrt.fx.passes.pass_utils:== Log pass <function fuse_permute_matmul at 0x7ff2e1c9c4c0> before/after graph to /tmp/tmp971_gpvf, before/after are the same = True, time elapsed = 0:00:01.290220
INFO:torch_tensorrt.fx.passes.pass_utils:== Log pass <function fuse_permute_linear at 0x7ff2e1c9c280> before/after graph to /tmp/tmp29tgm32h, before/after are the same = True, time elapsed = 0:00:00.609075
INFO:torch_tensorrt.fx.passes.pass_utils:== Log pass <function fix_clamp_numerical_limits_to_fp16 at 0x7ff2e1c9ca60> before/after graph to /tmp/tmp39p209q3, before/after are the same = True, time elapsed = 0:00:00.619397

Supported node types in the model:
acc_ops.conv2d: ((), {'input': torch.float16, 'weight': torch.float16})
acc_ops.batch_norm: ((), {'input': torch.float16, 'running_mean': torch.float16, 'running_var': torch.float16, 'weight': torch.float16, 'bias': torch.float16})
acc_ops.relu: ((), {'input': torch.float16})
acc_ops.max_pool2d: ((), {'input': torch.float16})
acc_ops.add: ((), {'input': torch.float16, 'other': torch.float16})
acc_ops.adaptive_avg_pool2d: ((), {'input': torch.float16})
acc_ops.flatten: ((), {'input': torch.float16})
acc_ops.linear: ((), {'input': torch.float16, 'weight': torch.float16, 'bias': torch.float16})

Unsupported node types in the model:

Got 1 acc subgraphs and 0 non-acc subgraphs
INFO:torch_tensorrt.fx.passes.lower_pass_manager_builder:Now lowering submodule _run_on_acc_0
INFO:torch_tensorrt.fx.lower:split_name=_run_on_acc_0, input_specs=[InputTensorSpec(shape=(-1, 3, 224, 224), dtype=torch.float16, device=device(type='cuda', index=0), shape_ranges=[((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224)), ((0, 3, 224, 224), (128, 3, 224, 224), (128, 3, 224, 224))], has_batch_dim=True)]
INFO:torch_tensorrt.fx.lower:Timing cache is used!
anaconda3/lib/python3.8/site-packages/torch_tensorrt/fx/fx2trt.py:200: UserWarning: Current platform doesn't support fast native fp16!
  warnings.warn("Current platform doesn't support fast native fp16!")
[03/19/2023-20:27:21] [TRT] [W] FP16 support requested on hardware without native FP16 support, performance will be negatively affected.
INFO:torch_tensorrt.fx.fx2trt:TRT INetwork construction elapsed time: 0:00:00.615370
INFO:torch_tensorrt.fx.fx2trt:Build TRT engine elapsed time: 0:00:30.495295
INFO:torch_tensorrt.fx.passes.lower_pass_manager_builder:Lowering submodule _run_on_acc_0 elapsed time 0:00:31.174103
== Start benchmark iterations
== End benchmark iterations
== Benchmark Result for: Configuration(batch_iter=50, batch_size=128, name='CUDA Eager', trt=False, jit=False, fp16=False, accuracy_rtol=-1)
BS: 128, Time per iter: 167.91ms, QPS: 762.34, Accuracy: None (rtol=-1)
== Benchmark Result for: Configuration(batch_iter=50, batch_size=128, name='TRT FP32 Eager', trt=True, jit=False, fp16=False, accuracy_rtol=0.001)
BS: 128, Time per iter: 163.44ms, QPS: 783.15, Accuracy: None (rtol=0.001)
== Benchmark Result for: Configuration(batch_iter=50, batch_size=128, name='TRT FP16 Eager', trt=True, jit=False, fp16=True, accuracy_rtol=0.01)
BS: 128, Time per iter: 163.63ms, QPS: 782.24, Accuracy: None (rtol=0.01)
